docs/checking-broken-links.md

```diff
@@ -1,34 +1,44 @@
 # Checking Broken Links
 
+## Introduction
+
+Checking for broken links is a science on its own. External webpages, internal sites and
+references to anchors can cause broken links in various ways. While it looks relatively easy to check
+for broken anchors, not many free tools provide this functionaliy. On the other hand side,
```

Comment:
s/functionaliy/functionality/

Reply:
s/On the other hand side,/On the other hand,/
or remove this phrase altogether - I am not sure that it adds value anyway.